dao/src/main/java/org/wso2/testgrid/dao/EntityManagerHelper.java

```diff
@@ -89,10 +95,34 @@ public static void closeEntityManager(String persistenceUnitName) {
      * @return entity manager factory with the given persistence unit name
      */
     private static EntityManagerFactory getEntityManagerFactory(String persistenceUnitName) {
+
+        Map<String, String> persistenceMap = new HashMap<String, String>();
         EntityManagerFactory entityManagerFactory = entityManagerFactoryMap.get(persistenceUnitName);
         if (entityManagerFactory == null) {
-            entityManagerFactory = Persistence.createEntityManagerFactory(persistenceUnitName);
-            entityManagerFactoryMap.put(persistenceUnitName, entityManagerFactory);
+            try {
+                //Get database info from testgrid-config.properties
+                String dbUrl = ConfigurationContext.getProperty(Constants.DB_URL);
+                String dbUser = ConfigurationContext.getProperty(Constants.DB_USER);
+                String dbUserPass = ConfigurationContext.getProperty(Constants.DB_USER_PASS);
```

Comment:
Can you provide a unit test for this code? As I see, the default configuration of config.properties is nowhere to be found now. 

I think we should create a default config file. If no config file is present in the TESTGRID_HOME, then we should place that default file.